docker/.gitpod.Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM gitpod/workspace-full
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update -qq \
+    && apt-get install -y \
+# x86_64 / generic packages
+      bash \
+      build-essential \
+      cmake \
+      git \
+      make \
+      python3 \
+      python3-pip \
+      tar \
+      unzip \
+      wget \ 
+# aarch64 packages
+      libffi-dev \
+      libssl-dev \
+      python3-dev \ 
+    && rm -rf /var/cache/apt/* /var/lib/apt/lists/*;
+
+# Needs to be installed as root
+RUN pip3 install adafruit-nrfutil
+
+RUN sudo chown -R gitpod /opt
+
+COPY docker/build.sh /opt/
+# Lets get each in a separate docker layer for better downloads
+# GCC
+RUN bash -c "source /opt/build.sh; GetGcc;"
+# NrfSdk
+RUN bash -c "source /opt/build.sh; GetNrfSdk;"
+# McuBoot
+RUN bash -c "source /opt/build.sh; GetMcuBoot;"
```

Comment:
This uses pip3 behind the scenes, but didn't error for me, so maybe this one is fine?

Reply:
Yeah I also see it not error, but I've wrapped all this bit as `USER root` anyway.
The `gitpod` user only really needs write access here if they want to run the Get... stuff during development (low likelihood?)